pom.xml

```diff
@@ -0,0 +1,391 @@
+  ~   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~   See the License for the specific language governing permissions and
+  ~   limitations under the License.
+  -->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.eclipse.ee4j</groupId>
+        <artifactId>project</artifactId>
+        <version>1.0.7</version>
+    </parent>
+
+    <groupId>jakarta.data</groupId>
+    <artifactId>jakarta-data-parent</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
+    <packaging>pom</packaging>
+
+    <name>Jakarta Data</name>
+    <description>Create the specification in Jakarta EE to help Jakarta EE developers create enterprise-grade applications applying Java® and data engine technologies. It helps them create scalable applications while maintaining low coupling with the underlying persistence technology.</description>
+    <url>www.eclipse.org/ee4j/data</url>
+    
+
+    <licenses>
+        <license>
+            <name>Eclipse Public License 2.0</name>
+            <url>https://www.eclipse.org/org/documents/epl-2.0/EPL-2.0.txt</url>
+        </license>
+    </licenses>
+
+    <scm>
+        <connection>scm:git:git://github.com/jakartaee/data.git</connection>
+        <developerConnection>scm:git:ssh://github.com:jakartaee/data.git</developerConnection>
+        <url>https://github.com/jakartaee/data</url>
+    </scm>
+
+    <developers>
+        <developer>
+            <name>Jakarta Data Developers</name>
+            <email>data-dev@eclipse.org</email>
+            <organization>Eclipse JakartaData</organization>
+            <organizationUrl>https://accounts.eclipse.org/mailing-list/data-dev</organizationUrl>
+        </developer>
+    </developers>
+
+    <properties>
+        <apache.rat.version>0.12</apache.rat.version>
+        <checkstyle.excludes></checkstyle.excludes>
+        <jacoco.maven.version>0.8.7</jacoco.maven.version>
+        <jakarta.annotation.version>1.3.5</jakarta.annotation.version>
+        <jakarta.enterprise.cdi.version>2.0.1</jakarta.enterprise.cdi.version>
+        <jakarta.json.bind.version>1.0.2</jakarta.json.bind.version>
+        <jakarta.json.version>1.1.6</jakarta.json.version>
+        <jakarta.data.version>${project.version}</jakarta.data.version>
+        <jakarta.validation.version>2.0.1</jakarta.validation.version>
+        <junit.version>5.8.2</junit.version>
+        <maven-javadoc-plugin.vesion>3.3.2</maven-javadoc-plugin.vesion>
+        <maven-source-plugin.version>3.2.1</maven-source-plugin.version>
+        <maven.checkstyle.plugin.version>3.1.2</maven.checkstyle.plugin.version>
+        <maven.compile.version>3.8.1</maven.compile.version>
+        <maven.compiler.source>11</maven.compiler.source>
```

Comment:
I'd rather use `maven.compiler.release` instead of `source/target`.

Reply:
I got this compile error:


```
[ERROR] COMPILATION ERROR : 
[INFO] -------------------------------------------------------------
[ERROR] Source option 5 is no longer supported. Use 6 or later.
[ERROR] Target option 1.5 is no longer supported. Use 1.6 or later.
[INFO] 2 errors 
[INFO] -------------------------------------------------------------
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for Jakarta Data 1.0.0-SNAPSHOT:
[INFO] 
[INFO] Jakarta Data ....................................... SUCCESS [  8.152 s]
[INFO] Jakarta Data API ................................... FAILURE [  1.256 s]
[INFO] Jakarta Data Specification ......................... SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  9.726 s
[INFO] Finished at: 2022-09-13T09:47:07+01:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.1:compile (default-compile) on project jakarta-data-api: Compilation failure: Compilation failure: 
[ERROR] Source option 5 is no longer supported. Use 6 or later.
[ERROR] Target option 1.5 is no longer supported. Use 1.6 or later.
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
[ERROR] 
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <args> -rf :jakarta-data-api
```

Reply:
> I got this compile error:

With what configuration?

Reply:
Maybe it's time to upgrade m-compiler-p. The latest is 3.10.1.

Reply:
This is strange as I'm getting:
> maven-compiler-plugin:3.8.1:compile

and it works fine, while in log above there is 
> org.apache.maven.plugins:maven-compiler-plugin:3.1:compile

3.1?

Reply:
Thanks, my mistake